src/api/Controllers/SettingsController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BDMS.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[AllowAnonymous]
+[Route("api/v{version:apiVersion}/[controller]")]
+public partial class SettingsController : ControllerBase
+{
+    private readonly IConfiguration configuration;
+
+    public SettingsController(IConfiguration serverConfiguration)
+    {
+        configuration = serverConfiguration;
+    }
+
+    [HttpGet("Auth")]
```

Comment:
PP: Routes generell kleinschreiben
```suggestion
    [HttpGet("auth")]
```